SETUP/tests/jsTests/formatPreviewTests.js

```diff
@@ -102,8 +102,8 @@ QUnit.module("Format preview test", function() {
     QUnit.test("unrecognised tag, u not enabled", function (assert) {
```

Comment:
Can we add some tests similar to the issue documented in the forums with the makePreview function?

Reply:
Good idea, I'll look into it.